src/hp_email.erl

```diff
@@ -25,35 +25,35 @@ send_html(ToEmails, FromEmail, Subject, Body) ->
   send(ToEmails, FromEmail, Subject, [{html, Body}]).
 
 send_email_verification(Email, Code) ->
-  From = <<"HolidayPing <holidayping@lambdaclass.com>">>,
-  Subject = <<"HolidayPing email confirmation">>,
-  Link = <<"https://holidayping.lambdaclass.com/register/confirm/code?code=", Code/binary,
+  From = <<"HolidayPinger <holidaypinger@lambdaclass.com>">>,
```

Comment:
The email should be something more like `reminder@holidaypinger.com`

Reply:
Changed in 749e248